notebooks/From FHIR to ML - Getting Started with the PHC Python SDK.ipynb

```diff
@@ -0,0 +1,2264 @@
+{
```

Comment:
<p>This is fine, but, boy I don't like the coloring of that confusion matrix. It really obscures what's happening there. The bright yellow is the most accurate cell since that's correctly predicting stage 2 as stage 2. But a 0 in an off-diagonal is a good thing. And... the absolute magnitude is not really important it's the proportion per-class. I wouldn't bother changing it, just making a comment. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lifeomic/phc-examples/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>